open-api/rest-catalog-open-api.yaml

```diff
@@ -2800,6 +3062,58 @@ components:
           additionalProperties:
             type: string
 
+    PrePlanTableRequest:
+      type: object
+      required:
+        - select
+        - options
+      properties:
+        select:
+          description: A list of the selected columns that user specifies during a query.
```

Comment:
"that user specifies during a query" seems very specific to query use case. I think we can just say "A list of the selected columns"